### ISSUE TITLE
getNamedChild always returns the first named child

### ISSUE THREAD

**Issue by ivar-rummelhoff** [bonede/tree-sitter-ng#20]
For some reason, 0 is used instead of the provided index in ```tree-sitter/src/main/c/org_treesitter_TSParser.c```:
```
JNIEXPORT jobject JNICALL Java_org_treesitter_TSParser_ts_1node_1named_1child
  (JNIEnv *env, jclass clz, jobject ts_node_object, jint index){
    TSNode ts_node = ts_node_from_obj(env, ts_node_object);
    TSNode child_ts_node = ts_node_named_child(ts_node, 0);
    return ts_node_to_obj(env, child_ts_node);
}
```

**Comment by bonede** [comment]
Hi @ivar-rummelhoff 

Thanks for reporting. I'll fix this in the next release.

**Comment by bonede** [comment]
fixed in `v0.22.6a` .
